Integrations/integration-Quinc.yml

```diff
@@ -0,0 +1,341 @@
+configuration:
+- display: ""
+  name: ServerName
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: ""
+  name: token
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: ""
+  name: scheme
+  defaultvalue: http
+  type: 15
+  required: true
+  options:
+  - http
+  - https
+script:
+  script: |-
+    import sys, os, requests, datetime, json
+    from requests.exceptions import HTTPError
+
+    # disable insecure warnings
+    requests.packages.urllib3.disable_warnings()
+
+    AUTH_HEADERS = { 'EnterpriseApiKey' : demisto.params()['token'], 'Accept': 'application/json', 'Content-Type':  'application/json'}
+    USE_SSL = not demisto.params().get('insecure', True)
+    QUINC_URL = demisto.params().get('scheme', 'http') + '://' + demisto.params().get('ServerName') + ':4443/'
+    JOBID_KEY = 'Accessdata.Job.ID'
+    JOBRESULT_KEY = 'Accessdata.Job.Result'
+    JOBSTATE_KEY = 'Accessdata.Job.State'
+    JOB_KEY = 'Accessdata.Job'
+
+    # Sends request to the server using the given method, url, headers and params
+    def send_request(method, url, body=None, headers=AUTH_HEADERS, params=None):
+        try:
+            res = requests.request(method, url, data=body, headers=headers, params=params, verify=USE_SSL)
+            res.raise_for_status()
+        except HTTPError, e:
+            err_json = res.json()
+            err_msg = ''
+            if 'message' in err_json:
+              err_msg = err_msg + 'Error: {0}.\n'.format(err_json['message'])
+            elif 'http_response' in err_json:
+              err_msg = err_msg + 'Error: {0}.\n'.format(err_json['http_response'])
+            if 'code' in err_json:
+              err_msg = err_msg + 'QRadar Error Code: {0}'.format(err_json['code'])
+            return_error(err_msg)
+        return res.json()
+
+    def SendQuincTestRequest():
+        url = 'api/v2/enterpriseapi/statuscheck';
+        res = send_request('GET', QUINC_URL + url)
+        return res.lower()
+
+    def QuincLegacyAgentSoftwareInventory(arg):
+        url = 'api/v2/enterpriseapi/agent/'+ arg['caseid'] +'/sofwareinventory';
+        data = {
+            'ips': { 'targets' : [arg['targetIP']]},
+            'SoftwareInvJob': {'Operation': 24 }
+        }
+        res = send_request('POST', QUINC_URL + url, json.dumps(data))
+        executeCommand('delContext', { key: JOBID_KEY });
+        return {
+            'Type': entryTypes['note'],
+            'Contents': res,
+            'ContentsFormat': formats['text'],
```

Comment:
does the `res` is text or JSON?

Reply:
It's a text,,
I've removed this code completely